RetailCoder.VBE/UI/Refactorings/Rename/RenameView.xaml.cs

```diff
@@ -5,6 +5,12 @@ public partial class RenameView
         public RenameView()
         {
             InitializeComponent();
+
+            base.Loaded += delegate
```

Comment:
You could remove the `delegate` keyword here, C# is clever enough to infer it :-)

The problem with anonymous handlers is that when the event provider is destroyed, there's no way to de-register the handler; if the event provider wasn't the same object that registers the event, this would be a memory leak (the handler is holding on to a reference that's otherwise eligible for garbage collection. Doesn't apply in this case, but definitely worth understanding.